root/static/scripts/common/i18n/expand2.js

```diff
@@ -386,7 +386,7 @@ export default function expand(source: string, args?: ?VarArgs): React.Node {
     result.length > 1
       ? React.createElement(React.Fragment, null, ...result)
       : result[0]
-  ) : null;
+  ) : '';
```

Comment:
Is there an example use case that requires backward compatibility?

Reply:
I think it makes no difference in practice. Some tests in root/static/scripts/tests/i18n/expand2.js fail if I revert this because there are strings like `'{x:%|}'` which can actually produce an empty result, but you wouldn't see those in actual code. I think it still makes sense to return an empty string in those cases, but I'll reword the commit message to make that more clear.